Documentation/HowToUseETW.md

```diff
@@ -0,0 +1,46 @@
+# Supported
+ETW logs are available on CoreCLR only currently.
```

Comment:
These instructions are windows only. I don't the mechanism to collect logs on Unix.